src/components/input/input.spec.js

```diff
@@ -346,6 +346,29 @@ describe('md-input-container directive', function() {
     expect(label.textContent).toEqual('some placeholder');
   });
 
+  it('should not create a floating label from a placeholder if md-no-float exists and is empty, or evaluates to true', function () {
```

Comment:
Perhaps I'm not seeing the corresponding code, but I think the description here is slightly misleading since this doesn't seem to test the case of `md-no-float exists and is empty`.

I think we may want to separate this into two tests if that's possible:

1. Test the empty case
2. Test the true case